File: go/wam/6302.go
const WamMdSyncdMutationsSummary = 6302 // channel: regular

type MdSyncdMutationsSummaryEvent struct {
	AppSessionId string `wam:"app_session_id"` // field 1
	CompanionSessionIds string `wam:"companion_session_ids"` // field 2
	IsInBootstrap bool `wam:"is_in_bootstrap"` // field 3
	LidMutations string `wam:"lid_mutations"` // field 4
	MutationBundle MUTATIONBUNDLETYPE `wam:"mutation_bundle"` // field 5
	MutationDirection MUTATIONDIRECTIONTYPE `wam:"mutation_direction"` // field 6
	PatchMac string `wam:"patch_mac"` // field 7
	RemoveMutations string `wam:"remove_mutations"` // field 8
	SeqNumber int64 `wam:"seq_number"` // field 9
	SetMutations string `wam:"set_mutations"` // field 10
	SnapshotMac string `wam:"snapshot_mac"` // field 11
	SyncdCollection SYNCDCOLLECTIONTYPE `wam:"syncd_collection"` // field 12
	SyncdKeyidKeyhash string `wam:"syncd_keyid_keyhash"` // field 13
}

File: ts/wam/6302.ts
export const WamMdSyncdMutationsSummary = 6302 as const

export interface MdSyncdMutationsSummaryEvent {
  /** field 1, wire `app_session_id` */
  appSessionId?: string
  /** field 2, wire `companion_session_ids` */
  companionSessionIds?: string
  /** field 3, wire `is_in_bootstrap` */
  isInBootstrap?: boolean
  /** field 4, wire `lid_mutations` */
  lidMutations?: string
  /** field 5, wire `mutation_bundle` */
  mutationBundle?: MUTATIONBUNDLETYPE
  /** field 6, wire `mutation_direction` */
  mutationDirection?: MUTATIONDIRECTIONTYPE
  /** field 7, wire `patch_mac` */
  patchMac?: string
  /** field 8, wire `remove_mutations` */
  removeMutations?: string
  /** field 9, wire `seq_number` */
  seqNumber?: number
  /** field 10, wire `set_mutations` */
  setMutations?: string
  /** field 11, wire `snapshot_mac` */
  snapshotMac?: string
  /** field 12, wire `syncd_collection` */
  syncdCollection?: SYNCDCOLLECTIONTYPE
  /** field 13, wire `syncd_keyid_keyhash` */
  syncdKeyidKeyhash?: string
}